src/DynamicData/Cache/ObservableCacheEx.cs

```diff
@@ -3592,24 +3592,6 @@ public static IObservable<T> FinallySafe<T>(this IObservable<T> source, Action f
         return sources.Combine(CombineOperator.Or);
     }
 
-    /// <summary>
-    /// Returns the page as specified by the pageRequests observable.
-    /// </summary>
-    /// <typeparam name="TObject">The type of the object.</typeparam>
-    /// <typeparam name="TKey">The type of the key.</typeparam>
-    /// <param name="source">The source.</param>
-    /// <param name="pageRequests">The page requests.</param>
-    /// <returns>An observable which emits change sets.</returns>
-    public static IObservable<IPagedChangeSet<TObject, TKey>> Page<TObject, TKey>(this IObservable<ISortedChangeSet<TObject, TKey>> source, IObservable<IPageRequest> pageRequests)
-        where TObject : notnull
-        where TKey : notnull
-    {
-        source.ThrowArgumentNullExceptionIfNull(nameof(source));
-        pageRequests.ThrowArgumentNullExceptionIfNull(nameof(pageRequests));
-
-        return new Page<TObject, TKey>(source, pageRequests).Run();
-    }
-
```

Comment:
If these operators truly are obsolete, I think we should just mark them as such, and remove them in the future, rather than removing them outright, right now.